documents/Libraries/stdlib/sx-glsl/mx_acescg_to_linear_color3.glsl

```diff
@@ -0,0 +1,5 @@
+void mx_acescg_to_linear_color3(vec3 _in, out vec3 result)
+{
+    vec4 outColor = vec4(_in, 0.);
+    result = (mat4(1.705079555511475, -0.1297005265951157, -0.02416634373366833, 0., -0.6242334842681885, 1.138468623161316, -0.1246141716837883, 0., -0.0808461606502533, -0.008768022060394287, 1.148780584335327, 0., 0., 0., 0., 1.) * outColor).rgb;
```

Comment:
It would be better to declare these as global constants but I think you'd need to turn them into arrays to allow this. Also you need to change the allow creating of additional constants in the implementation, so think you'd want to derive from SourceCode and add in an override for creating constants. Maybe something to look at after this.